src/qililab/instruments/quantum_machines/quantum_machines_cluster.py

```diff
@@ -354,52 +351,68 @@ def set_parameter_of_bus(self, bus: str, parameter: Parameter, value: float | st
 
             if parameter == Parameter.LO_FREQUENCY:
                 lo_frequency = float(value)
-                self._qm.octave.set_lo_frequency(element=bus, lo_frequency=lo_frequency)
+                # 1) Change settings:
```

Comment:
Is this really that confusing that needs these comments?

Reply:
Yeahh, Im not sure... also thought about removing them :)

 lemme make it simpler!

Reply:
Addressed! Check again how it looks now!

Reply:
Addressed @jordivallsq ! Check again how it looks now!